src/app/home/home.component.html

```diff
@@ -0,0 +1,91 @@
+<div class="background"></div>
+<mat-toolbar color="primary" class="header-nav">
+  <button mat-icon-button (click)="toggleNav()"><mat-icon>menu</mat-icon></button>
+  <span>Planet Learning</span>
+  <span class="toolbar-fill"></span>
+  <button mat-icon-button routerLink="/manager"><mat-icon>settings</mat-icon></button>
+  <button mat-icon-button [matMenuTriggerFor]="notificationMenu"><mat-icon>notifications</mat-icon></button>
+  <button mat-icon-button [matMenuTriggerFor]="languageMenu">
+    <img *ngIf="current_flag" src="assets/flags/{{current_flag}}.png" i18n-alt alt="{{current_lang}}" i18n-title

```

Comment:
Just noticed these `i18n-alt` and `i18n-title` tags won't work since the alt and title tags are set by interpolation.  We will need to update that.